pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -182,8 +183,18 @@ public interface FhirEtlOptions extends PipelineOptions {
 
   void setActivePeriod(String value);
 
+  @Description(
+      "Fetch only FHIR resources that were updated after the given timestamp."
+          + "The date format follows the dateTime format in the FHIR standard, without time-zone:\n"
+          + "https://www.hl7.org/fhir/datatypes.html#dateTime\n"
+          + "This feature is currently implemented only for HAPI JDBC mode.")
+  @Default.String("")
+  String getSince();
```

Comment:
Do we have any check in the code that ensures the format the user sets is correct?

Reply:
There is a TODO for this on line 94 of `JdbcFetchHapi.java`; the reason I am not much concerned about it is because HAPI+FHIR server do some sanity checking for us when we use this parameter to fetch number of resources (see changes in `FhirSearchUtil.java`). But still we need to do the TODO to make this more robust (in a future PR).

Reply:
Great!